src/components/Projects/ProjectView.tsx

```diff
@@ -53,28 +53,40 @@ function ProjectView({ project, onClose, isFullscreen = false }: Props) {
       },
     ]
   }, [project, showMilestonesTab, showUpdatesCta])
-  return (
-    <div>
-      {project && <ProjectSheetTitle project={project} onClose={onClose} isFullscreen={isFullscreen} />}
 
-      <BoxTabs className="ProjectView__Tabs">
-        <BoxTabs.Left>
-          {MENU_ITEMS.map((item, idx) => (
-            <BoxTabs.Tab key={idx} active={idx === viewIdx} onClick={() => setViewIdx(idx)}>
-              {t(item.labelKey)}
-              {!!item.showDot && <Dot />}
-            </BoxTabs.Tab>
-          ))}
-        </BoxTabs.Left>
-      </BoxTabs>
-      <div
-        className={classNames(
-          'ProjectView__ContentContainer',
-          isFullscreen && 'ProjectView__ContentContainer--fullscreen'
-        )}
-      >
+  return (
+    <div className="ProjectView">
+      {project && <ProjectViewTitle project={project} onClose={onClose} />}
```

Comment:
What do we show if there is no project?

Reply:
<img width="2213" alt="image" src="https://github.com/decentraland/governance-ui/assets/2858950/5fb8fd2d-3daa-4ef8-b452-595d9bdf33f2">